react/features/base/react/components/web/MeetingsList.js

```diff
@@ -88,20 +95,38 @@ export default class MeetingsList extends Component<Props> {
     constructor(props: Props) {
         super(props);
 
+        this._onClear = this._onClear.bind(this);
         this._onPress = this._onPress.bind(this);
         this._renderItem = this._renderItem.bind(this);
     }
 
+    _onClear: void => void;
+
+    /**
+     * Clear all history items and refresh page.
+     *
+     * @private
+     * @returns {void}
+     */
+    _onClear() {
+        window.localStorage.setItem('features/recent-list', []);
+        window.location.reload();
```

Comment:
I think we should handle this a little bit more gracefully. We already have an action that deletes a single entry from the list. Maybe you can create one for deleting all entries? I believe this will be auto synced with local storage and in the same time you won't need a reload to display the change.

Reply:
Yep, didn't see this action, it's only available on mobile app. I implemented a similar action to delete all the list (list entry delete).